src/MPS.jl

```diff
@@ -540,20 +541,29 @@ function mixed_canonize!(tn::AbstractMPO, orthog_center)
     return tn
 end
 
+LinearAlgebra.normalize(ψ::AbstractMPO; kwargs...) = normalize!(copy(ψ); kwargs...)
+LinearAlgebra.normalize(ψ::AbstractMPO, site::Site) = normalize!(copy(ψ), site)
```

Comment:
mmm maybe it's better if we write this method for `AbstractAnsatz`? that way we don't need to define it for all its subtypes.
Also, I think the first method is unnecessary because I believe (need to check) is already implemented for `AbstracTensorNetwork` or `AbstractQuantum`.

Reply:
No, `LinearAlgebra.normalize` it is not defined anywhere else.

Reply:
uuu then we should define it more generically for `AbstractQuantum` or so